redhat-developer/developers.redhat.com
# DEVELOPER-4583 [EPIC] Site cosmetic fixes

- [DEVELOPER-4505 PDF icon showing up on book CTA](https://issues.jboss.org/browse/DEVELOPER-4505)
- [DEVELOPER-4257 Commas missing from tags section on cheat sheets](https://issues.jboss.org/browse/DEVELOPER-4257)
- [DEVELOPER-4335 JBoss Awards - Page missing hero img](https://issues.jboss.org/browse/DEVELOPER-4335)
- [DEVELOPER-4334 Eclipse page missing header image](https://issues.jboss.org/browse/DEVELOPER-4334)
- [DEVELOPER-4333 Fix Devoxx4Kids page style problems](https://issues.jboss.org/browse/DEVELOPER-4333)

**STEPS TO REVIEW:**
1. Go to /books/building-reactive-microservices-java/ the CTA should not have a pdf icon, only text
2. Go to /cheat-sheets/containers/ the tags on the right side should be separated by a comma
3. Go to /devoxx4kids/ no images should be missing
4. Go to /eclipse/ no images should be missing
5. Go to /jboss-awards/ no images should be missing (except for headshot for Bill Burke and Charlie Nutter)

All of the removed files in this PR are old images no longer needed since they are in https://static.jboss.org/rhd/images/
